specs/jsonschema-core.md

```diff
@@ -1012,21 +1013,22 @@ resolution until runtime, at which point it is resolved each time it is
 encountered while evaluating an instance.
 
 Together with `$dynamicAnchor`, `$dynamicRef` implements a cooperative extension
-mechanism that is primarily useful with recursive schemas (schemas that
-reference themselves). The extension point is defined with `$dynamicAnchor` and
-only exhibits runtime dynamic behavior when referenced with `$dynamicRef`.
+mechanism that is primarily useful to extend recursive schemas, where
+`$dynamicRef` defines the extension point, and `$dynamicAnchor` defines the
+target.
```

Comment:
Unnecessary comma.

```suggestion
mechanism that is primarily useful to extend recursive schemas, where
`$dynamicRef` defines the extension point and `$dynamicAnchor` defines the
target.
```

Reply:
It's a compound sentence; the comma is necessary.